modules/rts/rts.types.ts

```diff
@@ -20,7 +20,7 @@ export type TSInfo = {
     retentionTime?: number,
     chunkCount?: number,
     chunkSize?: number,
-    duplicatePolicy?: boolean | null,
+    duplicatePolicy?: string | null,
```

Comment:
According to the [docs](https://redis.io/commands/ts.create/) duplication policy is not just a string, can you please add a type with the following values ->
```
DUPLICATE_POLICY policy

is policy for handling multiple samples with identical timestamps, with one of the following values:

    BLOCK: an error will occur for any out of order sample
    FIRST: ignore any newly reported value
    LAST: override with the newly reported value
    MIN: only override if the value is lower than the existing value
    MAX: only override if the value is higher than the existing value
    SUM: If a previous sample exists, add the new sample to it so that the updated value is equal to (previous + new). If no previous sample exists, set the updated value equal to the new value.

```

Reply:
I'm not too sure I follow, it seems like duplicate policy is one of the following strings you've already described in rts.types.ts line 85:

`export type TSDuplicatePolicyType = 'BLOCK' | 'FIRST' | 'LAST' | 'MIN' | 'MAX' | 'SUM';
`

Reply:
Okay cool, so we could use it :)